src/Components/Toast/Toast.jsx

```diff
@@ -3,22 +3,22 @@ import { useEffect } from "react";
 import "./Toast.css";
 
 export const Toast = () => {
-  const { toastTrigger, forToast, dispatch } = useEcom();
+  // const { toastTrigger, forToast, dispatch } = useEcom();
 
-  useEffect(() => {
-    const timeout = setTimeout(() => {
-      dispatch({ type: "TOAST_STATE_CLEAN" });
-    }, 1000);
-    return () => clearTimeout(timeout);
-  }, [forToast.trigger]);
+  // useEffect(() => {
+  //   const timeout = setTimeout(() => {
+  //     dispatch({ type: "TOAST_STATE_CLEAN" });
+  //   }, 1000);
+  //   return () => clearTimeout(timeout);
+  // }, [forToast.trigger]);
 
```

Comment:
suggestion - avoid pushing commented code

Reply:
The commented code is removed thanks for pointing it out